src/libstd/collections/hash/map.rs

```diff
@@ -667,28 +667,23 @@ impl<K, V, S> HashMap<K, V, S>
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
     pub fn reserve(&mut self, additional: usize) {
```

Comment:
I do want to wrap it up, but might as well do it properly. I'll take a look at this on Monday.